.github/workflows/mirror-libreoffice.yml

```diff
@@ -2,9 +2,40 @@ name: mirror-libreoffice
 
 on:
   workflow_dispatch:
+    inputs:
+      libreoffice_version:
+        description: The libreoffice version to mirror
+        required: true
 
 jobs:
   mirror:
+    strategy:
+      fail-fast: false
+      matrix:
+        package_type:
+          - deb
+          - rpm
     runs-on: ubuntu-latest
+    env:
+      FILENAME: LibreOffice_${{ inputs.libreoffice_version }}_Linux_x86-64_${{ matrix.package_type }}.tar.gz
     steps:
-      - run: echo for testing purposes
+      - name: Download libreoffice ${{ inputs.libreoffice_version }} ${{ matrix.package_type }}
+        run: wget https://downloadarchive.documentfoundation.org/libreoffice/old/${{ inputs.libreoffice_version }}/${{ matrix.package_type }}/x86_64/$FILENAME
+
+      - name: Calculate checksum (just for reference)
+        run: |
```

Comment:
Given we don't (and can't for now) check the integrity of the file using checksum we should at least check its signature before uploading it to our repo.